package-lock.json

```diff
@@ -791,7 +791,7 @@
         },
         "@types/websocket": {
             "version": "0.0.37",
-            "resolved": "https://registry.npmjs.org/@types/websocket/-/websocket-0.0.37.tgz",
```

Comment:
Revert changes to this file - seems like maybe this branch was so old before you cleaned your npm cache or something